packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -524,8 +560,8 @@ export const transformers = (model: ModelData): string[] => {
 			info.processor === "AutoTokenizer"
 				? "tokenizer"
 				: info.processor === "AutoFeatureExtractor"
-				  ? "extractor"
-				  : "processor";
+					? "extractor"
+					: "processor";
```

Comment:
cc @osanseviero this happens on each of my PRs on huggingface.js, I run linting using `prettier --write . `. My prettier version is 3.3.2. Anything I'm doing wrong?